src/components/flash/flash.js

```diff
@@ -124,7 +125,8 @@ class Flash extends React.Component {
      * @property dialogs
      * @type {Object}
      */
-    dialogs: {}
+    dialogs: {},
+    open: false
```

Comment:
[Minor] We should probably add some docs to `open` here, as all the other state and props have some.